### PR TITLE
Update content to match PowerShell 6 behavior.

### PR DESCRIPTION
**Pull request by JamesWTruher**
Break long lines along semantic line break locations

Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [x] The documented feature was introduced in version (6) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/JamesWTruher/PowerShell-Docs/blob/getcontentps6/reference/6/Microsoft.PowerShell.Management/Get-Content.md) | :x:Error | | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Management%2FGet-Content.md)
[]() | :x:Error | | [Details](#user-content-)

<a name="reference%2F6%2FMicrosoft.PowerShell.Management%2FGet-Content.md"></a>
### [reference/6/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/JamesWTruher/PowerShell-Docs/blob/getcontentps6/reference/6/Microsoft.PowerShell.Management/Get-Content.md)
 - **[Error]** Unable to load file: 6/Microsoft.PowerShell.Management/Get-Content.md via processor: AzureCliDocumentProcessor: :481:(1064) &#39;- Prior to PowerShell 6

&#39;
 Expect Heading

<a name=""></a>
### []()
 - **[Error]** Markdown.MAML.Transformer.HelpSchemaException: :481:(1064) &#39;- Prior to PowerShell 6

&#39;
 Expect Heading
   at Microsoft.DocAsCode.Build.Engine.SingleDocumentBuilder.GetInnerContexts(DocumentBuildParameters parameters, IEnumerable`1 processors, TemplateProcessor templateProcessor, IHostServiceCreator creator)
   at Microsoft.DocAsCode.Build.Engine.SingleDocumentBuilder.BuildCore(DocumentBuildParameters parameters)
   at Microsoft.DocAsCode.Build.Engine.SingleDocumentBuilder.Build(DocumentBuildParameters parameters)
   at Microsoft.DocAsCode.Build.Engine.DocumentBuilder.BuildCore(DocumentBuildParameters parameter, IMarkdownServiceProvider markdownServiceProvider, BuildInfo currentBuildInfo, BuildInfo lastBuildInfo)
   at Microsoft.DocAsCode.Build.Engine.DocumentBuilder.Build(IList`1 parameters, String outputDirectory)
   at Microsoft.DocAsCode.SubCommands.DocumentBuilderWrapper.BuildDocument(BuildJsonConfig config, TemplateManager templateManager, String baseDirectory, String outputDirectory, String pluginDirectory, String templateDirectory)
   at Microsoft.DocAsCode.SubCommands.DocumentBuilderWrapper.BuildDocument()

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C2%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201811022159459384-3237%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2BMqqSVYQ02ID6ticoyG9m65xpzPqqUOvYRrKgluJibI%3D&st=2018-11-02T22%3A07%3A59Z&se=2018-12-03T22%3A12%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/JamesWTruher/PowerShell-Docs/blob/getcontentps6/reference/6/Microsoft.PowerShell.Management/Get-Content.md) | :x:Error | | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Management%2FGet-Content.md)
[]() | :x:Error | | [Details](#user-content-)

<a name="reference%2F6%2FMicrosoft.PowerShell.Management%2FGet-Content.md"></a>
### [reference/6/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/JamesWTruher/PowerShell-Docs/blob/getcontentps6/reference/6/Microsoft.PowerShell.Management/Get-Content.md)
 - **[Error]** Unable to load file: 6/Microsoft.PowerShell.Management/Get-Content.md via processor: AzureCliDocumentProcessor: :481:(1064) &#39;- Prior to PowerShell 6

&#39;
 Expect Heading

<a name=""></a>
### []()
 - **[Error]** Markdown.MAML.Transformer.HelpSchemaException: :481:(1064) &#39;- Prior to PowerShell 6

&#39;
 Expect Heading
   at Microsoft.DocAsCode.Build.Engine.SingleDocumentBuilder.GetInnerContexts(DocumentBuildParameters parameters, IEnumerable`1 processors, TemplateProcessor templateProcessor, IHostServiceCreator creator)
   at Microsoft.DocAsCode.Build.Engine.SingleDocumentBuilder.BuildCore(DocumentBuildParameters parameters)
   at Microsoft.DocAsCode.Build.Engine.SingleDocumentBuilder.Build(DocumentBuildParameters parameters)
   at Microsoft.DocAsCode.Build.Engine.DocumentBuilder.BuildCore(DocumentBuildParameters parameter, IMarkdownServiceProvider markdownServiceProvider, BuildInfo currentBuildInfo, BuildInfo lastBuildInfo)
   at Microsoft.DocAsCode.Build.Engine.DocumentBuilder.Build(IList`1 parameters, String outputDirectory)
   at Microsoft.DocAsCode.SubCommands.DocumentBuilderWrapper.BuildDocument(BuildJsonConfig config, TemplateManager templateManager, String baseDirectory, String outputDirectory, String pluginDirectory, String templateDirectory)
   at Microsoft.DocAsCode.SubCommands.DocumentBuilderWrapper.BuildDocument()

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C2%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201811022216383424-3237%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=fWzRcRybUv6Sl1qaEIGUrLd9a81wVCTs0crz0jCQq98%3D&st=2018-11-02T22%3A36%3A27Z&se=2018-12-03T22%3A41%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/6/Microsoft.PowerShell.Management/Get-Content.md [hunk 1]
```diff
@@ -401,30 +438,57 @@ Accept wildcard characters: False
...
+> Beginning in PowerShell 6.0, the type of the Encoding parameter changed from
+> `FileSystemCmdletProviderEncoding` to `System.Text.Encoding`, and **Byte** is no longer
+> a valid option for the `-Encoding` parameter.
+> You can use the `-AsByteStream` parameter to indicate that the content should be read and
+> output as a byte stream.
+
+- PowerShell 6
+
+```yaml
+Type: System.Text.Encoding
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Management/Get-Content.md]: Having two yaml blocks is not allowed by PlatyPS. You should only document the v6.x information in this v6 document.

Comment (JamesWTruher) [reply reference/6/Microsoft.PowerShell.Management/Get-Content.md]: will do

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/JamesWTruher/PowerShell-Docs/blob/getcontentps6/reference/6/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-6&branch=pr-en-us-3237) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C6%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201811052357249050-3237%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=zLR7mi20t7cnBU0jEYr7nuijSE1E1i74GQ45vDHlSfA%3D&st=2018-11-06T00%3A06%3A34Z&se=2018-12-07T00%3A11%3A34Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**